doc/developers/maintainer.rst

```diff
@@ -54,24 +97,19 @@ Making a release
 
    - Update the release date in whats_new.rst
 
-   - Edit the doc/index.rst to change the 'News' entry of the front page.
+   - Edit the doc/templates/index.html to change the 'News' entry of the front page.
 
    - Note that these changes should be made in master and cherry-picked into
-     the release branch.
+     the release branch, only before the final release.
 
 2. On the branch for releasing, update the version number in
    sklearn/__init__.py, the ``__version__`` variable by removing ``dev*`` only
-   when ready to release.
-   On master, increment the verson in the same place (when branching for
-   release).
-
-3. Create the tag and push it::
-
-    $ git tag -a 0.999
-
-    $ git push git@github.com:scikit-learn/scikit-learn.git --tags
+   when ready to release. On master, increment the version in the same place
+   (when branching for release). This means while we're in the release
+   candidate period, the latest stable is two versions behind the master
+   branch, and one one.
```

Comment:
One one was a racehorse two two was one two. One one one one race. Two two one one two. 

Not sure what one one is doing here.